### ISSUE TITLE
heidisql.com: detection

### ISSUE THREAD

**Issue by 3xploiton3** [uBlockOrigin/uAssets#20430]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] This is NOT a [YouTube](https://www.github.com/uBlockOrigin/uAssets/issues/19976), [Facebook](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [Twitch](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report.
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [X] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://www.heidisql.com/download.php?download=installer`

### Category

detection

### Description

![image](https://github.com/uBlockOrigin/uAssets/assets/19517680/ce43fc3b-9c17-4131-bf73-5baf79da7de1)

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.52.2
Chromium: 118
filterset (summary):
 network: 111087
 cosmetic: 113769
 scriptlet: 26914
 html: 0
listset (total-discarded, last-updated):
 added:
  https://adblockid.pages.dev/adblockid.adfl.txt: 10413-0, now
  https://adblockid.pages.dev/adblockid_plus.adfl.txt: 80-4, now
  easylist-annoyances: 4158-34, now
  easylist-notifications: 2858-81, now
  fanboy-cookiemonster: 48791-219, now
  fanboy-social: 16155-10, now
  ublock-annoyances: 5684-79, now
  ublock-cookies-easylist: 430-430, now
 default:
  user-filters: 29-0, never
  easylist: 72321-21, now
  easyprivacy: 32858-151, now
  plowe-0: 3720-1073, now
  ublock-badware: 7421-140, 25m
  ublock-filters: 35176-274, 24m
  ublock-privacy: 641-8, now
  ublock-quick-fixes: 239-184, now
  ublock-unbreak: 2135-36, 23m
  urlhaus-1: 11555-0, now
filterset (user): [array of 29 redacted]
trustedset:
 added: [array of 56 redacted]
switchRuleset:
 added: [array of 2 redacted]
userSettings:
 cloudStorageEnabled: true
 suspendUntilListsAreLoaded: true
hiddenSettings: [none]
supportStats:
 allReadyAfter: 1375 ms (selfie)
 maxAssetCacheWait: 458 ms
popupPanel:
 blocked: 1
 network:
  googlesyndication.com: 1
 extended:
  ##ins.adsbygoogle
  ##div.cookie-consent
  ##+js(no-setTimeout-if, increaserev)
  ##+js(abort-current-script, $, adblock)
```
</details>

**Comment by JobcenterTycoon** [comment]
Yes for Firefox is a manual click needed because the filter causing a error in Firefox

`Uncaught TypeError: cannot declare global binding 'floovy': property must be configurable or both writable and enumerable`

**Comment by JobcenterTycoon** [comment]
I just mean the autodownload fails on FF but a user can click on the download link to get the download
